src/pages/studyView/table/tables.module.scss

```diff
@@ -13,4 +13,5 @@
 
 .body {
```

Comment:
i think we should avoid a class named "body" as it's too easily confused with <body>.  lets make this more specific